templates/cli/lib/commands/generic.js.twig

```diff
@@ -122,8 +151,51 @@ const logout = new Command("logout")
             sdk: client
         })
 
-        globalConfig.setCookie("");
-        success()
+        globalConfig.removeSession(accountId);
+    } catch (e) {
+        error('Unable to log out, removing locally saved session information')
+    }
+    globalConfig.removeSession(accountId);
+}
+
+const logout = new Command("logout")
+    .description(commandDescriptions['logout'])
+    .configureHelp({
+        helpWidth: process.stdout.columns || 80
+    })
+    .action(actionRunner(async () => {
+        const sessions = globalConfig.getSessions();
+        const current = globalConfig.getCurrentSession();
+
+        if (current === '') {
+            return;
+        }
+        if (sessions.length === 1) {
+            await deleteSession(current);
+            success();
+
+            return;
+        }
+
+        const answers = await inquirer.prompt(questionsLogout);
+
+        if (answers.accounts) {
+            for (let accountId of answers.accounts) {
+                globalConfig.setCurrentSession(accountId);
+                await deleteSession(accountId);
```

Comment:
we are setting the current session to an accountID and deleting that specific one? Which means all new requests will throw 401 ?

Reply:
We're setting the current session to the last account in line using the `leftSession` variable right after.